requirements/frozen.txt

```diff
@@ -279,6 +287,7 @@ PyYAML==5.1
 pyzmq==18.0.1
 qtconsole==4.4.3
 redfish-client==0.1.0
+redis==3.3.10
```

Comment:
added as part of dynaconf requirements . https://github.com/rochacbruno/dynaconf/blob/master/setup.py